src/main/java/seedu/address/model/policy/PolicyList.java

```diff
@@ -70,7 +84,7 @@ public static PolicyList createNewPolicyList() {
      * else returns null.
      */
     public Policy getPolicy(PolicyName name) {
-        return policies.get(name.policyName);
+        return policies.get(name.value);
     }
 
```

Comment:
Is createNewPolicyList() being used? If it is, we can add it into the PolicyListTest.java